docs/topics/native/native-spm.md

```diff
@@ -0,0 +1,192 @@
+[//]: # (title: Swift Package export setup)
+
+Kotlin/Native output for an Apple target can be consumed as an SPM dependency.
+For example, if your Kotlin Multiplatform project has an iOS target, you may want to make the binary available
+as a dependency to iOS developers working on native Swift projects in Xcode. Using KMP tooling, you can provide
+an artifact that would seamlessly integrate with Xcode experience.
+
+This guide shows how to do this using the [XCFramework building](multiplatform-build-native-binaries.md#build-xcframeworks)
+capability of Kotlin Gradle.
+
+## Prepare locations for the files to be uploaded
+
+To make your framework available to be consumed, you will need to upload two files:
+* A ZIP archive of the XCFramework itself should be uploaded to a convenient file storage with direct access (for example,
+S3, GitHub Releases, or a Maven repository). Choose the option that is easiest to integrate into your workflow.
```

Comment:
A couple of concerns about the repository list:

* Is "GitHub Releases" a separate product name? Do we need to underline the releases here? If we want to stick to the official names, I think it should be "Amazon S3" as well
* I think it would be great to both simplify the names and make it consistent with the Maven item:

>  for example, an S3, GitHub, or Maven repository